esp32_s3_lcd_ev_board/include/bsp/esp32_s3_lcd_ev_board.h

```diff
@@ -215,13 +260,13 @@ esp_err_t bsp_audio_init(const i2s_std_config_t *i2s_config, i2s_chan_handle_t *
  *      - ESP_OK:               On success
  *      - ESP_ERR_INVALID_ARG:  Invalid GPIO number
  */
-esp_err_t bsp_audio_poweramp_enable(const bool enable);
+esp_err_t bsp_audio_poweramp_enable(bool enable);
```

Comment:
Add deprecated or delete.

Reply:
I add info in comment to indicate it's deprecated.